stories/busy-indicator/busy-indicator.stories.js

```diff
@@ -68,6 +69,37 @@ export const contrastMode = () => `<div style="display:flex;justify-content:cent
 contrastMode.parameters = {
     docs: {
         iframeHeight: 200,
-        storyDescription: 'The busy indicator also comes in contrast mode and displays white dots against a dark background. To apply contrast mode, add <code>contrast</code> into the element i.e. <code>fd-busy-indicator--m contrast</code>.'
+        storyDescription:
+            'The busy indicator also comes in contrast mode and displays white dots against a dark background. To apply contrast mode, add <code>contrast</code> into the element i.e. <code>fd-busy-indicator--m contrast</code>.'
+    }
+};
+export const LabelledIndicator = () => `<div style="text-align: center">
+    <div class="fd-busy-indicator fd-busy-indicator--l" aria-hidden="false" aria-label="Loading">
+        <div class="fd-busy-indicator--circle-0"></div>
+        <div class="fd-busy-indicator--circle-1"></div>
+        <div class="fd-busy-indicator--circle-2"></div>
+		<span class="fd-busy-indicator--label fd-form-label">loading data...</span>
+    </div>`;
+
+LabelledIndicator.parameters = {
+    docs: {
+        iframeHeight: 200,
+        storyDescription:
+            'The standard busy indicator animates a sequence of cascading dots expanding and shrinking in a loop. The component comes with a label/message to be displayed when the data is being loaded. <code>fd-busy-indicator--label</code> is used to display the message.'
+    }
+};
+export const ExtendedIndicator = () => `<div style="text-align: center">
+    <div class="fd-busy-indicator--l fd-busy-indicator--enhanced-indicator" aria-hidden="false" aria-label="Loading">
+        <div class="fd-busy-indicator--circle-0"></div>
+        <div class="fd-busy-indicator--circle-1"></div>
+        <div class="fd-busy-indicator--circle-2"></div>
+		<span class=" fd-busy-indicator--enhanced-indicator-text fd-busy-indicator--label fd-form-label">loading data...</span>
+    </div>`;
+
+ExtendedIndicator.parameters = {
```

Comment:
The modifier is called `enhanced-indicator` but the example name is `Extended Indicator`.  Can this be made consistent?

Reply:
fixed